### PR TITLE
UI Developer's Guide part VI.

### PR DESCRIPTION
**Pull request by martinpovolny**
More docs.

Ping @mzazrivec, @himdel

### CODE REVIEW

File: ui/listnav.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+### Listnavs
+
+Listnavs are the left parts of the non-explorer screens. They typicaly contain information about entities and their relationships organized into accordions.
```
Comment (dclarizio) [inline ui/listnav.md]: `typicaly` -> `typically`

File: ui/menus.md [hunk 2]
```diff
@@ -15,13 +15,13 @@ The main menu styling supports 3 level, that is 2 levels of menu section. Menus
...
- * text, 
- * icon and 
+Menu sections looks like the example above. It consists of:
+ * Identifier,
+ * text,
+ * icon and
  * items.
 
-Identifier is the same as the [RBAC feature](rbac_features.md) that is checked when desicison is make whether the particular section is displayed.
+Identifier is the same as the [RBAC feature](rbac_features.md) that is checked when decision is make whether the particular section is displayed.
```
Comment (dclarizio) [inline ui/menus.md]: `when decision is make whether the particular section is displayed` ->
`when a decision is made as to whether or not the particular section is displayed`

File: ui/menus.md [hunk 3]
```diff
@@ -56,3 +56,21 @@ Rbac\_feature is a hash that is passed to `role_allows?` call when decision is b
...
 
 Type can be either :default or :big\_iframe. We stick with the :default type which is default. The :big\_iframe type is used for [simple UI plugins](simple_ui_plugins.md).
+
+#### Remembering last clicks
+
+Each menu section remembers last GET request that was done "under" that menu
+section. Then if a user clicks a menu section (not a menu item) the last
+remembered GET request's URL is opened so that the user lands where he last was "under" that menu section.
+
+This requeres a relationship between controller and menu section.
```
Comment (dclarizio) [inline ui/menus.md]: requires

File: ui/simple_ui_plugins.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+### Simple UI plugins
+
+The UI has a very simple mechanism for plugging in menu sections and items.
+
+There's a number of limitations:
```
Comment (dclarizio) [inline ui/simple_ui_plugins.md]: There are a number of limitations:

File: ui/topology.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+### Topology
+
+Topologies are graphical representations of relationships between entities such
+ast VMs, Hosts, Networks, Container, Providers and more.
```
Comment (dclarizio) [inline ui/topology.md]: `ast` -> `as`

File: ui/simple_ui_plugins.md (hunk 4 above)
Comment (martinpovolny) [inline ui/simple_ui_plugins.md]: done

File: ui/menus.md (hunk 3 above)
Comment (martinpovolny) [inline ui/menus.md]: done

File: ui/topology.md (hunk 5 above)
Comment (martinpovolny) [inline ui/topology.md]: done

**Comment by martinpovolny** [comment]
@dclarizio : all typos fixed. Thx!

**Comment by isimluk** [comment]
@martinpovolny you can use `:set spell` in your favorite editor of choice
:snowboarder:

**Pull request merged**